index.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <script src="https://kit.fontawesome.com/138cf4be1d.js" crossorigin="anonymous"></script>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <title>Akpuru Solomon Website</title>
+</head>
+<body>
+  <div id="container">
+   <header class="header">
+      <div class="topnav" id="myTopnav">
+         <a id="bars" onclick="" href="#"><i class="fa fa-bars"></i></a>
+        <a id="mess" href="#"><img src="./image/Union.png" alt="Message Icon" /></a>
+        <a id="logo" href="#">Akpuru Solomon</a>
+        <a href="#works" class="active">Portfolio</a>
+        <a href="#about">About</a>
+        <a href="#contact">Contact</a>
+    
+      </div>
+    </header>
+    <div class="pop-up pop" id="pop-menu">
+  <button id="close" onclick="closeWindow()"><img class="img-button" src="./image/cancel-btn.png" alt="Close Window Button" /></button>
+  <ul class="navigation">
+    <li class="nav-tools first-tool hide" ><a class="the-nav1" href="#work">Portfolio</a></li>
+    <li class="nav-tools hide"><a class="the-nav2" href="#about_section">About</a></li>
+    <li class="nav-tools hide"><a class="the-nav3" href="#contact">Contact</a></li>
+  </ul>
```

Comment:
- [ ] Kindly be noticed that in this portfolio design you should make the menu height a full page and also you should make the font and the background like the Figma design.

Figma Design | Your Design
------------------ | ------------------
![Screenshot from 2022-07-26 22-05-37](https://user-images.githubusercontent.com/33752735/181102864-aabc3ae5-248d-4c52-b903-0c982aec4a7b.png) | ![Screenshot from 2022-07-26 22-06-01](https://user-images.githubusercontent.com/33752735/181102900-23f67c08-cc78-4d01-976f-607d41679e30.png)